modules/standard/RecordParser.chpl

```diff
@@ -94,9 +192,15 @@ class RecordReader {
     } while(once);
   }
 
-  // An internal function that we use all our user visible code.
-  // When called with no arguments it does no checking abut block boundaries
-  // HDFS specific when offst and len are given. Checks for block boundaries for parallel file IO
+  /*
+     An internal function that we use all our user visible code.  When called
+     with no arguments it does no checking abut block boundaries.
+     
+     HDFS specific when offst and len are given. Checks for block boundaries
+     for parallel file IO
```

Comment:
Not totally sure... I think Tim might have meant that it's handling for offst and len
is only used within a context like HDFS. But since this comment doesn't even
appear in the chpldoc, I'm not too worried about it right now :)